### ISSUE TITLE
[Improvement] Split the trigger of the sounds & animations on the client from the broadcast to the other players

### ISSUE THREAD

**Issue by PalaNolho** [gsimon2/dramatic-rolls#11]
It seems that the sounds and confetti are only being played when the roll is public.

This seems to have been done because, if we roll in private, we don't want players to know when a crit is played and when that happens the sounds are being broadcast to the players.

Based on this, I think it would make sense to split the play if the sounds and animations from the actual broadcast of the sounds to the other players.

We could have a flag in the config (checked y default) saying 
> play animations & sounds on Public rolls only [ X ] 

if unchecked, the animations & sounds should play for self rolls on the client regardless of the type of roll

Another useful info would be to have a "static flag (aka read-only) just to clarify the fact that the sounds & animations will only be seen by other players on public rolls. Now, for this one there is a catch... 
- when the DM rolls, it makes sense that he only broadcasts when he is rolling public
- but when a player rolls for the GM (privately or blind) it makes sense that the animations still play OR, at least give the option to the GM to choose if he want to see them our not.

Hope that it' helps
Keep up the good work 👍

**Comment by gsimon2** [comment]
Working on this now. Will have to test it with all compatible mods / systems before creating a release. Thank you for taking such excellent notes on this feature, helped immensely after coming backing to it after a couple months. I apologize for the delay.

**Comment by PalaNolho** [comment]
Pleasure :)

**Comment by gsimon2** [comment]
I am happy to say that I finally got around to this (mostly). As of version `1.5.0` there is now a setting that allows effects to be played only locally for non-public rolls. These rolls only trigger effects for the rolling user. Public rolls will still have the effects broadcast to every player. I did not address 

> but when a player rolls for the GM (privately or blind) it makes sense that the animations still play OR, at least give the option to the GM to choose if he want to see them our not.

 as the logic got very fuzzy. Instead the are handled the same. If this is a strongly desired feature I can leave this issue open and try to tackle it in the future. Thanks for posting!
